adafruit_ble_broadcastnet.py

```diff
@@ -56,12 +57,16 @@ def broadcast(measurement, *, broadcast_time=0.1, extended=False):
         _sequence_number = (_sequence_number + 1) % 256
 
 
-device_address = "{:02x}{:02x}{:02x}{:02x}{:02x}{:02x}".format(  # pylint: disable=invalid-name
-    *reversed(
-        list(_ble._adapter.address.address_bytes)  # pylint: disable=protected-access
+# This line causes issues with Sphinx, so we won't run it in the CI
+if "GITHUB_ACTION" not in os.environ and "READTHEDOCS" not in os.environ:
```

Comment:
I steered you a little wrong here. There is no `os.environ` in CircuitPython, only in regular CPython, so this will throw an exception. So I'd add `hasattr(os, "environ") to the beginning of the `if`. Also, could you test importing this library on CircuitPython

Reply:
Ah, ok. I'll test that as soon as I make that change